packages/angular/README.md

```diff
@@ -0,0 +1,237 @@
+<p align="center">
+  <a href="https://sentry.io" target="_blank" align="center">
+    <img src="https://sentry-brand.storage.googleapis.com/sentry-logo-black.png" width="280">
+  </a>
+  <br />
+</p>
+
+# Official Sentry SDK for Angular
+
+## Links
+
+- [Official SDK Docs](https://docs.sentry.io/quickstart/)
+- [TypeDoc](http://getsentry.github.io/sentry-javascript/)
```

Comment:
There should be a link to more specific Angular docs, like https://docs.sentry.io/platforms/javascript/angular/ or wherever the new docs will be.